src/main/resources/application.properties

```diff
@@ -1,2 +1 @@
-spring.main.allow-bean-definition-overriding=true
-spring.data.mongodb.uri=${mongoconnect}
\ No newline at end of file
+spring.main.allow-bean-definition-overriding=true
```

Comment:
What does this do?

Reply:
it resolved a conflicts between mongodb and spring libraries. I will investigate if it's still necessary

Reply:
has been removed now as it's unnecessary